addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxNotificationPlayer.java

```diff
@@ -225,18 +225,23 @@ private void playNotification() throws InterruptedException, SqueezeBoxTimeoutEx
     private void restorePlayerState() {
         logger.debug("Restoring player state");
 
-        // Resume playing save playlist item if player wasn't stopped.
+        // Resume playing save playlist item.
         // Note that setting the time doesn't work for remote streams.
-        if (!playerState.isStopped()) {
-            logger.debug("Resuming last item playing");
-            squeezeBoxServerHandler.playPlaylistItem(mac, playerState.getPlaylistIndex());
-            squeezeBoxServerHandler.setPlayingTime(mac, playerState.getPlayingTime());
-        } else if (!playerState.isPlaying()) {
-            logger.debug("Pausing the player");
-            squeezeBoxServerHandler.pause(mac);
-        } else {
-            logger.debug("Stopping the player");
-            squeezeBoxServerHandler.stop(mac);
+        squeezeBoxServerHandler.playPlaylistItem(mac, playerState.getPlaylistIndex());
```

Comment:
Would the consequence of this order of statements, be that we first restart the playlist items and then quickly after stop or pause it? How does this sound / work in practice?

Reply:
When you tell the LMS to go to a playlist item, it automatically starts playing it. To the best of my knowledge, that's the way the LMS API works, unfortunately. To avoid an audible sound, the volume is set to 0 before loading the playlist, then the volume is restored after the stop/pause operation.

Reply:
And where is that logic hiding out?

Reply:
It's hiding here 
https://github.com/pgfeller/openhab2-addons/blob/825ec1eb0a0eef3b03ae358abdba36e53ab81a44/addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxNotificationPlayer.java#L76 

and here
https://github.com/pgfeller/openhab2-addons/blob/825ec1eb0a0eef3b03ae358abdba36e53ab81a44/addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxNotificationPlayer.java#L250

Reply:
Would it make more sense to have that more integrated, it does not need to be part of this PR. But if feel strange that the preparation of cleaning up is disconnected from the cleaning up.

Reply:
Yes, I was thinking the same thing. I'll take care of it in another PR. Thanks.